modules/dragster/src/test/ts/atomic/datatransfer/DataTransferCloneTest.ts

```diff
@@ -0,0 +1,92 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { KAssert } from '@ephox/katamari-assertions';
+import { PlatformDetection } from '@ephox/sand';
+import { assert } from 'chai';
+
+import { cloneDataTransfer, createDataTransfer, getDragImage } from 'ephox/dragster/datatransfer/DataTransfer';
+import { Event, getEvent, setDragstartEvent } from 'ephox/dragster/datatransfer/Event';
+import { getMode, Mode, setMode, setReadOnlyMode, setReadWriteMode } from 'ephox/dragster/datatransfer/Mode';
+
+describe('atomic.dragster.datatransfer.DataTransferCloneTest', () => {
```

Comment:
Not sure these tests are technically atomic since it uses the dom. The intent at least in the past was that atomic tests where tests that was just plain JS and could in theory be executed without a browser context like in node.

Reply:
Good to know, made the dataTransfer tests browser tests.